tests/CoreBundle/Service/DirectoryTest.php

```diff
@@ -127,7 +127,7 @@ public function testFind()
     public function testFindOutputEscaping()
     {
         $this->config->shouldReceive('get')->once()->with('ldap_directory_campus_id_property')->andReturn('campusId');
-        $filter= '(|(sn=a\**)(givenname=a\**)(mail=a\**)(campusId=a\**))';
+        $filter= '(&(|(sn=a\2a*)(givenname=a\2a*)(mail=a\2a*)(campusId=a\2a*)))';
```

Comment:
It's picky, but do we care if there's a space between `$filter` and `=` here?

Reply:
I think we do yes, but since it was like that already I'm going to fix it in a separate PR.